web-console/4.0-designs/overview/overview.md

```diff
@@ -0,0 +1,87 @@
+# Overview
+
+## Collapsed list view
+
+### General
+
+![list view](img/listview.png)
+* The project overview may be viewed in list view or topology view. It will default to show the list view. Users can toggle between the two views by using the view selector icons at the top right corner of the page.
+* Users may filter resources by name by using the filter bar in the top right.
+* The collapsed list view of the project overview will group items by app label (for example, `app=foo`) by default. Users may group by a different label by using the `Group by label` dropdown selector. The selector will be populated with existing labels.
+  * If no resources have labels, items will be shown in a regular list view listed alphabetically by resource type and then name and the `Group by label` dropdown will be hidden.
+* On hover, the row will be highlighted (#EDF8FF) and the icon fa-angle-right (pf-blue-400, #0088CE) will appear at the end of the row. When a row is selected (#DEF3FF), the side panel will open with a summary of that resource's overview, pushing the content over to the left.
+* If a user is not in the context of a selected project, the user will be shown an [empty state](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/empty-states/empty-states) until they select a project from the project dropdown.
+![empty state](img/ES_overview.png)
+
+### List columns
+
+* The first column will include resource badges and names. Users can redirect to a resource's details page by clicking the resource name.
+* The second column will display status updates and other relevant notifications and events such as crash looping containers and deployment failures. Users can hover on a status or notification to see more information.
+* The third column will be a summary of the item's metrics, including memory, CPU, and network. If metrics are not enabled, this column will be blank. If metrics are enabled but are not currently available, users will see "--" in place of the values.
+* The fourth column will show how many pods have deployed for that item. Users can click on the pods link to be directed to the resource's pods page.
+
+### Route URL
+
+![route](img/route.png)
+* Users may navigate to the application by clicking the route URL link (pf-blue-400, #0088CE) across from the application name. The icon fa-external-link (pf-blue-400, #0088CE) will be to the right of the URL to indicate that the link redirects users to an external location and opens a new browser tab.
+* Users may edit which route URL is displayed by clicking the black (pf-black-900, #292E34) pficon-edit icon. A popover will open where users may set their default route from a list of their routes. Users may also copy route URLs from this popover. Users may click Cancel or ‘x’ or click anywhere else on the screen to close the popover, or click Save to confirm their selection.
+  * The default route will be set only for the current user.
```

Comment:
I still wonder about the usefulness of this. Was this limitation called out when we reviewed the design for picking a route with stakeholders?

Reply:
@spadgett yeah I thought customers have been asking for this but maybe @serenamarie125 can chime in since she's probably been in the most customer meetings.

Reply:
@spadgett usefulness of having a default route at all?  We have definitely heard this request from customers in the past.

Reply:
Right, I'm just not sure this solves what we've been asked for. Being able to set my own default route that only I see only in this one browser seems really limited. And with the new overview design, it also depends on what label I've selected in the group by dropdown. As soon as I group by the a different label, the visible route for each group will change again.